### ISSUE TITLE
Showing currentPosition on screen

### ISSUE THREAD

**Issue by Sargnec** [hyochan/react-native-audio-recorder-player#250]
### Version of react-native-audio-recorder-player
"react-native-audio-recorder-player": "^2.5.0",
### Platforms you faced the error (IOS or Android or both?)
Android
### Question
Im setting playTime in onStartPlay as you can see below. But this gets so slow like when audio ends playTime is not where it should be. How can I show currentPosition on screen while playing the audio but it shouldn't be delayed like I'm doing

### Steps to reproduce the behavior
 ```
 const onStartPlay = (item, index) => {
        let path = item.voiceFile.slice(7)
        const msg = audioRecorderPlayer.startPlayer(path);
        setStarted({status:true,index:index})
        audioRecorderPlayer.addPlayBackListener((e) => {
            if (e.current_position === e.duration) {
                audioRecorderPlayer.stopPlayer(path);
                audioRecorderPlayer.removePlayBackListener(path);
            }
            //Play time
            let newArray = [...sendedVoices]
            newArray[index].playTime = audioRecorderPlayer.mmssss(Math.floor(e.current_position))
            setSendedVoices(newArray)
            return;
        })
    };
```

**Comment by hyochan** [comment]
I might have to add additional functionality to reduce the communication type between the js thread and native.

**Comment by hyochan** [comment]
@Sargnec Could you try `Player.setSubscriptionDuration(500)` and reduce the communication rate?

**Comment by hyochan** [comment]
I think this will be resolved in `3.0.0`.
